packages/db/test/utils/index.coffee

```diff
@@ -1,18 +1,113 @@
-{db} = require '../../src/utils'
-
-describe 'utils', ->
+{db} = require '../test'
+utils = require '../../src/utils'
+{command, escape, jdbc} = utils.db
 
+describe "db.utils", ->
+  
+  describe "escape", ->
+  
+    it 'backslashes', ->
+      escape('\\').should.eql '\\\\'
+  
+    it 'double quotes', ->
+      escape('"').should.eql '\\"'
+  
+    it 'backslashes and double quotes', ->
+      query = 'SELECT * FROM my_db WHERE name = "John\\\'s"'
+      expected = 'SELECT * FROM my_db WHERE name = \\"John\\\\\'s\\"'
+      escape(query).should.eql expected
+      
   describe 'jdbc', ->
 
       it 'get default port', ->
-        db.jdbc('jdbc:mysql://localhost/my_db').should.eql
+        jdbc('jdbc:mysql://localhost/my_db').should.eql
           engine: 'mysql',
           addresses: [ { host: 'localhost', port: 3306 } ],
           database: 'my_db'
-        db.jdbc('jdbc:postgresql://localhost/my_db').should.eql
+        jdbc('jdbc:postgresql://localhost/my_db').should.eql
           engine: 'postgresql',
           addresses: [ { host: 'localhost', port: 5432 } ],
           database: 'my_db'
 
       it 'get database', ->
-        db.jdbc('jdbc:mysql://master3.ryba:3306/my_db?a_param=true').database.should.eql 'my_db'
+        jdbc('jdbc:mysql://master3.ryba:3306/my_db?a_param=true').database.should.eql 'my_db'
+          
+  describe 'command', ->
+    
+    it 'invalid engine', ->
+      () -> command
+        engine: 'invalid_engine'
+      .should.throw 'Unsupported engine: "invalid_engine"'
+      
+    describe "using engine: mariadb", ->
+      return unless db.mariadb?
```

Comment:
why this check, the tests below don't interact with a database, there is no need to filter them

Reply:
I added in case the configuration of the database is removed from the `test.coffee` file, but yes, is an additional no necessary check.